packages/amplify-category-hosting/__tests__/lib/S3AndCloudFront/helpers/file-uploader.test.js

```diff
@@ -80,19 +84,27 @@ describe('cloudfront-manager', () => {
     },
   };
 
-  beforeAll(() => {
+  beforeAll(async () => {
     fs.createReadStream = jest.fn(() => {
       return {};
     });
     mime.lookup = jest.fn(() => {
       return 'text/plain';
     });
+    await fileUploader.run(mockContext, 'mockDistributionFolder');
```

Comment:
the "functionality under test" should not be in `beforeAll` block, that block is reserved for test setup.
Readability of the tests (i.e. right call in right block) should take precedence over duplication.

Reply:
Done ada487f2515a0f9edef8eaeaa85a0e6c7c627024